crates/fuzzing/wasm-spec-interpreter/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = ["The Wasmtime Project Developers"]
+description = "A Rust-to-OCaml wrapper for the WebAssembly specification interpreter"
+name = "wasm-spec-interpreter"
+version = "0.1.0"
+publish = false
+edition = "2018"
+license = "Apache-2.0 WITH LLVM-exception"
+
+[dependencies]
+ocaml-interop = { version = "0.8", optional = true }
```

Comment:
Should this be a build dependency since you don't want it if ocaml is deemed not available by build.rs?

Reply:
No, it's a dependency used at runtime.